package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "linkify-plus-plus",
   "description": "Based on Linkify Plus. Turn plain text URLs into links.",
-  "version": "10.0.0",
+  "version": "10.0.1",
```

Comment:
Do not modify unrelated part.

BTW the version number should be bumped by `npm version` command instead of editing manually.

Reply:
back to 10.0.0